operators/go/submariner_controller.go

```diff
@@ -0,0 +1,181 @@
+package submariner
+
+import (
+	"context"
+
+	submarinerv1alpha1 "github.com/submariner-operator/submariner-operator/pkg/apis/submariner/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+
+	//apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
nit: delete this?

Reply:
Good catch, fixed in the latest, thanks!

Reply:
This is long since done